components/brave_wallet/browser/eth_data_parser.cc

```diff
@@ -74,8 +74,10 @@ bool GetTransactionInfoFromData(const std::string& data,
   }
   if (!IsValidHexString(data))
     return false;
-  if (data.length() < 10)
-    return false;
+  if (data.length() < 10) {
```

Comment:
It's still a valid transaction so we should return true here. It just doesn't call a contract.